src/gauge/constructs/naikConstructs.h

```diff
@@ -33,7 +33,7 @@ __device__ GSU3<floatT> inline naik3LinkUp(gaugeAccessor<floatT> gAcc, gSite sit
 }
 /*
 template<class floatT,size_t HaloDepth>
-__device__ GSU3<floatT> inline naik3LinkDn(gaugeAccessor<floatT> gAcc, gSite site, int mu) {
+DEVICE GSU3<floatT> inline naik3LinkDn(gaugeAccessor<floatT> gAcc, gSite site, int mu) {
```

Comment:
this may also fail

Reply:
this is a find-replace error, good catch